src/formatter-settings/utils.ts

```diff
@@ -148,3 +157,65 @@ export function parseProfile(document: vscode.TextDocument): ProfileContent {
         supportedProfileSettings,
     }
 }
+
+export async function downloadFile(settingsUrl: string, extensionVersion: string, targetPath?: string): Promise<string> {
+    if (targetPath) {
+        await fse.ensureDir(path.dirname(targetPath));
+        if (await fse.pathExists(targetPath)) {
+            await fse.remove(targetPath);
+        }
+    }
+    return await new Promise((resolve: (res: string) => void, reject: (e: Error) => void): void => {
```

Comment:
Can we use some third-party library to download the file, like `Axios`

Reply:
Sure. Will try `Axios` then.